app/src/main/java/com/eveningoutpost/dexdrip/Services/WixelReader.java

```diff
@@ -70,34 +69,32 @@ public static void sStop() {
         singleton = null;
     }
 
-    public static boolean IsConfigured(Context ctx) {
+    public static boolean isConfigured(Context ctx) {
         SharedPreferences prefs = PreferenceManager.getDefaultSharedPreferences(ctx);
         String recieversIpAddresses = prefs.getString("wifi_recievers_addresses", "");
-        if(recieversIpAddresses == null || recieversIpAddresses.equals("") ) {
+        if (recieversIpAddresses == null || recieversIpAddresses.equals("")) {
             return false;
         }
         return true;
     }
 
-    public static boolean almostEquals( TransmitterRawData e1, TransmitterRawData e2)
-    {
-        if (e1 == null || e2==null) {
+    public static boolean almostEquals(TransmitterRawData e1, TransmitterRawData e2) {
+        if (e1 == null || e2 == null) {
             return false;
         }
         // relative time is in ms
-        if ((Math.abs(e1.CaptureDateTime - e2.CaptureDateTime) < 120 * 1000 ) &&
+        if ((Math.abs(e1.CaptureDateTime - e2.CaptureDateTime) < 120 * 1000) &&
                 (e1.TransmissionId == e2.TransmissionId)) {
             return true;
         }
         return false;
     }
 
- // last in the array, is first in time
-    public static List<TransmitterRawData> Merge2Lists(List<TransmitterRawData> list1 , List<TransmitterRawData> list2)
-    {
-        List<TransmitterRawData> merged = new LinkedList <TransmitterRawData>();
+    // last in the array, is first in time
+    public static List<TransmitterRawData> merge2Lists(List<TransmitterRawData> list1, List<TransmitterRawData> list2) {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The method merge2Lists() has an NPath complexity of 49](https://www.codacy.com/app/matthias-granberry/xDrip-Experimental/file/1313443989/issues/source?bid=2735053&fileBranchId=2735067#l94)